h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -2010,4 +2010,21 @@ public static int occurrenceCount(byte[] array, byte element) {
 
     return cnt;
   }
+
+  public static String toJavaDoubleArray(double[] array) {
```

Comment:
these array serialisation methods are usually part of genmodel, ArrayUtils is more for array operations, I dont mind keeping this here, but maybe some PojoUtils class would be a better place

Reply:
fixed